From 82f7d9c8cf211c1e29f239b68dfb4789bb7f1f6f Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@golden-gryphon.com>
Date: Fri, 25 Apr 2014 22:37:45 -0700
Subject: [PATCH 2/2] [master]: Fix FTBFS with gcc-4.8.

Signed-off-by: Manoj Srivastava <srivasta@golden-gryphon.com>
---
 Configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Configure b/Configure
index 63fabdd..5bf2b97 100755
--- a/Configure
+++ b/Configure
@@ -6383,7 +6383,7 @@ do
 		-e 's/^.*"\(.*\)".*\$/'\$filebase'$_o: \1/' \\
 		-e 's|: \./|: |' \\
 		-e 's|: *$|: '\$srcfile'|' | \\
-	$grep -v '^#' | $sort | $uniq
+	$grep -v '^#' | grep -v '<.*>' |$sort | $uniq
 done
 exit 0
 EOS
-- 
2.0.0.rc0

